db/schema.rb

```diff
@@ -214,6 +214,15 @@
     t.index ["petition_id", "constituency_id"], name: "idx_constituency_petition_journal_uniqueness", unique: true
   end
 
+  create_table "contacts", force: :cascade do |t|
+    t.bigint "signature_id", null: false
```

Comment:
Hmm, just noticed that some stuff is integer and some stuff is bigint - we should migrate to the latter but we can do that in another PR.